webapp/src/main/webapp/vue-apps/search-activity/components/SearchActivityCard.vue

```diff
@@ -188,6 +189,9 @@ export default {
     activityTitle() {
       return this.excerptHtml || this.$t('search.activity.no.title.label');
     },
+    activityTitleText() {
+      return $('<div />').html(this.activityTitle).text();
```

Comment:
Please use `this.$utils.htmlToText`. JQuery must not be used anymore since it's deprecated.